docker/conf/psinode/scripts/run-admin-dashboards

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# This script assumes that psinode exposes port 8079 over http
```

Comment:
Correct, the http psinode image automatically starts these tools using supervisor, rather than with a manually-run script.

Reply:
I feel like the scripts should only be copied into the images that they are useful for to reduce potential confusion.  Also, if this script is run manually, would it make sense to give it a name that starts with `psi` to make it easier to identify as one of our tools and also easier to find by tab completion in the terminal?

Reply:
Okay, I renamed the script to psi-run-https-admin-dashboards to address both concerns.
Technically it's still a valid script for this image, because someone could bind mount their own psinode config that uses https into the container.